Add Conflict to bruno-bin

bupd opened the pull request:
## Summary
The `bruno` package was missing the conflict to `bruno-bin`. I have added conflicts for the bruno package.

Thanks

bupd committed:
add conflicts bruno-bin

Signed-off-by: bupd <bupdprasanth@gmail.com>

bupd committed:
update pkgrel

Signed-off-by: bupd <bupdprasanth@gmail.com>

vbouchaud commented:
Hi there

As I wrote [here](https://github.com/usebruno/bruno/issues/31#issuecomment-2833608249), the three bruno related PKGBUILD already conflict with each other.

They all mention the `provides` keyword with the value `bruno` and they also all mention the `conflicts` keyword with the same value `bruno`. Therefore they are already conflicting with each other and you will not be able to install one of these without uninstalling the others beforehand.

Please see [`provides`](https://wiki.archlinux.org/title/PKGBUILD#provides) and [`conflicts`](https://wiki.archlinux.org/title/PKGBUILD#conflicts) documentation on archlinux's wiki if my explanation was not clear enough.

Thank you for your contribution. I'm closing this since it already works as intended.

bupd commented:
Okay good, I suggested to keep it as explicit to avoid this problem and save some bandwidth.

fail fast fail early. Hope this makes sense


```shell
❯ yay -S bruno-bin
AUR Explicit (1): bruno-bin-2.2.0-1
:: PKGBUILD up to date, skipping download: bruno-bin
  1 bruno-bin                        (Installed) (Build Files Exist)
==> Packages to cleanBuild?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
  1 bruno-bin                        (Installed) (Build Files Exist)
==> Diffs to show?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 

==> Making package: bruno-bin 2.2.0-1 (Mon 28 Apr 2025 10:40:43 PM IST)
==> Retrieving sources...
  -> Found bruno_2.2.0_amd64_linux.deb
  -> Found LICENSE-2.2.0
==> WARNING: Skipping verification of source file PGP signatures.
==> Validating source files with sha256sums...
    bruno_2.2.0_amd64_linux.deb ... 
Passed
    LICENSE-2.2.0 ... Passed
:: (1/1) Parsing SRCINFO: bruno-bin
==> Making package: bruno-bin 2.2.0-1 (Mon 28 Apr 2025 10:40:44 PM IST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found bruno_2.2.0_amd64_linux.deb
  -> Found LICENSE-2.2.0
==> Validating source files with sha256sums...
    bruno_2.2.0_amd64_linux.deb ... Passed
    LICENSE-2.2.0 ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting bruno_2.2.0_amd64_linux.deb with bsdtar
==> Sources are ready.
 -> bruno-bin-2.2.0-1 already made -- skipping build
==> Making package: bruno-bin 2.2.0-1 (Mon 28 Apr 2025 10:40:45 PM IST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Sources are ready.


loading packages...
warning: bruno-bin-2.2.0-1 is up to date -- reinstalling
warning: bruno-bin-debug-2.2.0-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Packages (2) bruno-bin-2.2.0-1  bruno-bin-debug-2.2.0-1

Total Installed Size:  356.19 MiB
Net Upgrade Size:        0.00 MiB

:: Proceed with installation? [Y/n] y
(2/2) checking keys in keyring                                                                                    [####################################################################] 100%
(2/2) checking package integrity                                                                                  [####################################################################] 100%
(2/2) loading package files                                                                                       [####################################################################] 100%
(2/2) checking for file conflicts                                                                                 [####################################################################] 100%
(2/2) checking available disk space                                                                               [####################################################################] 100%
:: Processing package changes...
(1/2) reinstalling bruno-bin                                                                                      [####################################################################] 100%
(2/2) reinstalling bruno-bin-debug                                                                                [####################################################################] 100%
:: Running post-transaction hooks...
(1/3) Arming ConditionNeedsUpdate...
(2/3) Updating icon theme caches...
(3/3) Updating the desktop file MIME type cache...
~                                                                                                                                                                                        10s
❯ yay -S bruno    
AUR Explicit (1): bruno-2.0.1-1
Sync Make Dependency (1): asar-3.4.1-1
Sync Dependency (1): electron33-33.4.8-2
:: PKGBUILD up to date, skipping download: bruno
  1 bruno                            (Build Files Exist)
==> Packages to cleanBuild?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
  1 bruno                            (Build Files Exist)
==> Diffs to show?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 

==> Making package: bruno 2.0.1-1 (Mon 28 Apr 2025 10:40:59 PM IST)
==> Retrieving sources...
  -> Found bruno-2.0.1.tar.gz
  -> Found com.usebruno.app.Bruno.desktop
==> WARNING: Skipping verification of source file PGP signatures.
==> Validating source files with sha256sums...
    bruno-2.0.1.tar.gz ... Passed
    com.usebruno.app.Bruno.desktop ... Passed
:: Remove make dependencies after install? [y/N] :: (1/1) Parsing SRCINFO: bruno
resolving dependencies...
looking for conflicting packages...

Packages (2) asar-3.4.1-1  electron33-33.4.8-2

Total Download Size:    77.87 MiB
Total Installed Size:  259.23 MiB

:: Proceed with installation? [Y/n] y
:: Retrieving packages...
 asar-3.4.1-1-any                                                                      86.4 KiB   144 KiB/s 00:01 [####################################################################] 100%
 electron33-33.4.8-2-x86_64                                                            77.8 MiB  25.2 MiB/s 00:03 [####################################################################] 100%
 Total (2/2)                                                                           77.9 MiB  25.0 MiB/s 00:03 [####################################################################] 100%
(2/2) checking keys in keyring                                                                                    [####################################################################] 100%
(2/2) checking package integrity                                                                                  [####################################################################] 100%
(2/2) loading package files                                                                                       [####################################################################] 100%
(2/2) checking for file conflicts                                                                                 [####################################################################] 100%
(2/2) checking available disk space                                                                               [####################################################################] 100%
:: Processing package changes...
(1/2) installing asar                                                                                             [####################################################################] 100%
(2/2) installing electron33                                                                                       [####################################################################] 100%
Optional dependencies for electron33
    kde-cli-tools: file deletion support (kioclient5)
    pipewire: WebRTC desktop sharing under Wayland [installed]
    qt5-base: enable Qt5 with --enable-features=AllowQt [installed]
    gtk4: for --gtk-version=4 (GTK4 IME might work better on Wayland) [installed]
    trash-cli: file deletion support (trash-put)
    xdg-utils: open URLs with desktop’s default (xdg-email, xdg-open) [installed]
:: Running post-transaction hooks...
(1/2) Arming ConditionNeedsUpdate...
(2/2) Updating the desktop file MIME type cache...
==> Making package: bruno 2.0.1-1 (Mon 28 Apr 2025 10:41:05 PM IST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found bruno-2.0.1.tar.gz
  -> Found com.usebruno.app.Bruno.desktop
==> Validating source files with sha256sums...
    bruno-2.0.1.tar.gz ... Passed
    com.usebruno.app.Bruno.desktop ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting bruno-2.0.1.tar.gz with bsdtar
==> Starting prepare()...
mkdir: created directory '/home/bupd/.cache/yay/bruno/src/bruno-2.0.1/.nvm'
'/home/bupd/.cache/yay/bruno/src/bruno-2.0.1/.nvm/nvm.sh' -> '/usr/share/nvm/nvm.sh'
'/home/bupd/.cache/yay/bruno/src/bruno-2.0.1/.nvm/nvm-exec' -> '/usr/share/nvm/nvm-exec'
Found '/home/bupd/.cache/yay/bruno/src/bruno-2.0.1/.nvmrc' with version <v22.11.0>
Downloading and installing node v22.11.0...
Downloading https://nodejs.org/dist/v22.11.0/node-v22.11.0-linux-x64.tar.xz...
###################################################################################################################################################################################### 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v22.11.0 (npm v10.9.0)
Creating default alias: default -> v22.11.0
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'httpsnippet@3.0.9',
npm warn EBADENGINE   required: { node: '^14.19.1 || ^16.14.2 || ^18.0.0 || ^20.0.0' },
npm warn EBADENGINE   current: { node: 'v22.11.0', npm: '10.9.0' }
npm warn EBADENGINE }
npm warn deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm warn deprecated rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser
npm warn deprecated npmlog@6.0.2: This package is no longer supported.
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated glob@8.1.0: Glob versions prior to v9 are no longer supported
npm warn deprecated are-we-there-yet@3.0.1: This package is no longer supported.
npm warn deprecated har-validator@5.1.5: this library is no longer supported
npm warn deprecated gauge@4.0.4: This package is no longer supported.
npm warn deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
npm warn deprecated boolean@3.2.0: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated electron-notarize@1.2.2: Please use @electron/notarize moving forward.  There is no API change, just a package name change
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated are-we-there-yet@2.0.0: This package is no longer supported.
npm warn deprecated npmlog@5.0.1: This package is no longer supported.
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated gauge@3.0.2: This package is no longer supported.

> prepare
> husky install

husky - HUSKY env variable is set to 0, skipping install

added 1979 packages, and audited 1991 packages in 2m

221 packages are looking for funding
  run `npm fund` for details

3 vulnerabilities (2 moderate, 1 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
npm notice
npm notice New major version of npm available! 10.9.0 -> 11.3.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.3.0
npm notice To update run: npm install -g npm@11.3.0
npm notice
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'httpsnippet@3.0.9',
npm warn EBADENGINE   required: { node: '^14.19.1 || ^16.14.2 || ^18.0.0 || ^20.0.0' },
npm warn EBADENGINE   current: { node: 'v22.11.0', npm: '10.9.0' }
npm warn EBADENGINE }
npm warn skipping integrity check for git dependency ssh://git@github.com/electron/node-gyp.git
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported

added 72 packages, changed 20 packages, and audited 2063 packages in 2m

221 packages are looking for funding
  run `npm fund` for details

3 vulnerabilities (2 moderate, 1 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'httpsnippet@3.0.9',
npm warn EBADENGINE   required: { node: '^14.19.1 || ^16.14.2 || ^18.0.0 || ^20.0.0' },
npm warn EBADENGINE   current: { node: 'v22.11.0', npm: '10.9.0' }
npm warn EBADENGINE }

added 14 packages, removed 7 packages, changed 3 packages, and audited 2070 packages in 18s

222 packages are looking for funding
  run `npm fund` for details

3 vulnerabilities (2 moderate, 1 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
==> Sources are ready.
==> Making package: bruno 2.0.1-1 (Mon 28 Apr 2025 10:45:01 PM IST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Starting build()...

> build:graphql-docs
> npm run build --workspace=packages/bruno-graphql-docs


> @usebruno/graphql-docs@0.1.0 build
> rollup -c


src/index.ts → dist/cjs/index.js, dist/esm/index.js...
created dist/cjs/index.js, dist/esm/index.js in 5s

dist/esm/index.d.ts → dist/index.d.ts...
created dist/index.d.ts in 13ms

> build:bruno-query
> npm run build --workspace=packages/bruno-query


> @usebruno/query@0.1.0 prebuild
> npm run clean


> @usebruno/query@0.1.0 clean
> rimraf dist


> @usebruno/query@0.1.0 build
> rollup -c


src/index.ts → dist/cjs/index.js, dist/esm/index.js...
created dist/cjs/index.js, dist/esm/index.js in 1.5s

dist/esm/index.d.ts → dist/index.d.ts...
created dist/index.d.ts in 17ms

> build:bruno-common
> npm run build --workspace=packages/bruno-common


> @usebruno/common@0.1.0 prebuild
> npm run clean


> @usebruno/common@0.1.0 clean
> rimraf dist


> @usebruno/common@0.1.0 build
> rollup -c


src/index.ts → dist/cjs/index.js, dist/esm/index.js...
created dist/cjs/index.js, dist/esm/index.js in 2.2s

dist/esm/index.d.ts → dist/index.d.ts...
created dist/index.d.ts in 6ms

> @usebruno/js@0.12.0 sandbox:bundle-libraries
> node ./src/sandbox/bundle-libraries.js

Circular dependency: ../../node_modules/chai/lib/chai.js -> ../../node_modules/chai/lib/chai/utils/index.js -> ../../node_modules/chai/lib/chai/utils/addProperty.js -> ../../node_modules/chai/lib/chai.js
Circular dependency: ../../node_modules/chai/lib/chai.js -> ../../node_modules/chai/lib/chai/utils/index.js -> ../../node_modules/chai/lib/chai/utils/addMethod.js -> ../../node_modules/chai/lib/chai.js
Circular dependency: ../../node_modules/chai/lib/chai.js -> ../../node_modules/chai/lib/chai/utils/index.js -> ../../node_modules/chai/lib/chai/utils/overwriteProperty.js -> ../../node_modules/chai/lib/chai.js
Circular dependency: ../../node_modules/chai/lib/chai.js -> ../../node_modules/chai/lib/chai/utils/index.js -> ../../node_modules/chai/lib/chai/utils/overwriteMethod.js -> ../../node_modules/chai/lib/chai.js
Circular dependency: ../../node_modules/chai/lib/chai.js -> ../../node_modules/chai/lib/chai/utils/index.js -> ../../node_modules/chai/lib/chai/utils/addChainableMethod.js -> ../../node_modules/chai/lib/chai.js
Circular dependency: ../../node_modules/chai/lib/chai.js -> ../../node_modules/chai/lib/chai/utils/index.js -> ../../node_modules/chai/lib/chai/utils/overwriteChainableMethod.js -> ../../node_modules/chai/lib/chai.js

> build:web
> npm run build --workspace=packages/bruno-app


> @usebruno/app@2.0.0 build
> rsbuild build -m production

  Rsbuild v1.1.13

● web ━━━━━━━━━━━━━━━━━━━━━━━━━ (100%) emitting after emit                                                                                                                                   ready   Built in 14.7 s (web)

  File (web)                                                       Size         Gzip    
  dist/index.html                                                  0.55 kB      0.28 kB
  dist/static/font/inter-vietnamese-100-normal.03f887b7.woff2      4.9 kB       4.9 kB
  dist/static/font/inter-vietnamese-400-normal.de4fc44f.woff2      5.0 kB       5.0 kB
  dist/static/font/inter-vietnamese-900-normal.e639b65a.woff2      5.0 kB       5.1 kB
  dist/static/font/inter-vietnamese-300-normal.ad9ef503.woff2      5.1 kB       5.1 kB
  dist/static/font/inter-vietnamese-200-normal.d4b1139a.woff2      5.1 kB       5.1 kB
  dist/static/font/inter-vietnamese-600-normal.9d518599.woff2      5.1 kB       5.1 kB
  dist/static/font/inter-vietnamese-500-normal.7c0a695f.woff2      5.1 kB       5.1 kB
  dist/static/font/inter-vietnamese-800-normal.922627b4.woff2      5.2 kB       5.2 kB
  dist/static/font/inter-vietnamese-700-normal.c48feea2.woff2      5.2 kB       5.2 kB
  dist/static/font/inter-greek-ext-400-normal.f8992900.woff2       5.2 kB       5.3 kB
  dist/static/font/inter-greek-ext-100-normal.3d999e5e.woff2       5.3 kB       5.3 kB
  dist/static/font/inter-greek-ext-300-normal.7b467784.woff2       5.4 kB       5.4 kB
  dist/static/font/inter-greek-ext-200-normal.64dcccdd.woff2       5.4 kB       5.4 kB
  dist/static/font/inter-greek-ext-500-normal.5fe403a5.woff2       5.4 kB       5.4 kB
  dist/static/font/inter-greek-ext-900-normal.3c30e30e.woff2       5.4 kB       5.4 kB
  dist/static/font/inter-greek-ext-700-normal.411652e2.woff2       5.4 kB       5.4 kB
  dist/static/font/inter-greek-ext-600-normal.d05f940f.woff2       5.4 kB       5.5 kB
  dist/static/font/inter-greek-ext-800-normal.95d04d18.woff2       5.5 kB       5.5 kB
  dist/static/font/inter-vietnamese-100-normal.c33529d0.woff       6.6 kB       6.6 kB
  dist/static/font/inter-vietnamese-900-normal.5af8132c.woff       6.6 kB       6.6 kB
  dist/static/font/inter-vietnamese-400-normal.1411920a.woff       6.7 kB       6.7 kB
  dist/static/font/inter-vietnamese-200-normal.d1738c75.woff       6.8 kB       6.8 kB
  dist/static/font/inter-vietnamese-500-normal.c5840ea0.woff       6.8 kB       6.8 kB
  dist/static/font/inter-vietnamese-300-normal.b4574483.woff       6.8 kB       6.8 kB
  dist/static/font/inter-vietnamese-700-normal.26a4f6eb.woff       6.8 kB       6.8 kB
  dist/static/font/inter-vietnamese-800-normal.6d6c1e5d.woff       6.8 kB       6.8 kB
  dist/static/font/inter-vietnamese-600-normal.8b0a74d0.woff       6.8 kB       6.8 kB
  dist/static/font/inter-greek-ext-100-normal.1423dba6.woff        7.1 kB       7.0 kB
  dist/static/font/inter-greek-ext-400-normal.f196e968.woff        7.1 kB       7.1 kB
  dist/static/font/inter-greek-ext-900-normal.44e5e662.woff        7.2 kB       7.1 kB
  dist/static/font/inter-greek-ext-300-normal.fb5ad981.woff        7.2 kB       7.2 kB
  dist/static/font/inter-greek-ext-200-normal.28c1da10.woff        7.2 kB       7.2 kB
  dist/static/font/inter-greek-ext-500-normal.34eb831d.woff        7.3 kB       7.2 kB
  dist/static/font/inter-greek-ext-800-normal.1b2bb72c.woff        7.3 kB       7.2 kB
  dist/static/font/inter-greek-ext-700-normal.fa338c24.woff        7.3 kB       7.2 kB
  dist/static/font/inter-greek-ext-600-normal.a46b5cba.woff        7.3 kB       7.2 kB
  dist/static/font/inter-cyrillic-100-normal.dc90b237.woff2        7.6 kB       7.6 kB
  dist/static/font/inter-cyrillic-400-normal.547767ef.woff2        7.7 kB       7.7 kB
  dist/static/font/inter-greek-100-normal.4601ceba.woff2           7.7 kB       7.8 kB
  dist/static/font/inter-cyrillic-200-normal.3f3a159e.woff2        7.8 kB       7.8 kB
  dist/static/font/inter-cyrillic-900-normal.aa7ab13c.woff2        7.8 kB       7.8 kB
  dist/static/font/inter-cyrillic-300-normal.7335a360.woff2        7.8 kB       7.8 kB
  dist/static/font/inter-greek-900-normal.22ac87d8.woff2           7.8 kB       7.9 kB
  dist/static/font/inter-greek-400-normal.573bacd1.woff2           7.8 kB       7.9 kB
  dist/static/font/inter-greek-300-normal.14123a0c.woff2           7.9 kB       7.9 kB
  dist/static/font/inter-greek-500-normal.947d4ab3.woff2           7.9 kB       7.9 kB
  dist/static/font/inter-cyrillic-500-normal.5ec9103b.woff2        7.9 kB       7.9 kB
  dist/static/font/inter-greek-200-normal.d64aa322.woff2           7.9 kB       7.9 kB
  dist/static/font/inter-greek-800-normal.fe474d33.woff2           7.9 kB       8.0 kB
  dist/static/font/inter-greek-700-normal.a094cf2b.woff2           7.9 kB       8.0 kB
  dist/static/font/inter-greek-600-normal.ee808ffe.woff2           8.0 kB       8.0 kB
  dist/static/font/inter-cyrillic-800-normal.0e80493d.woff2        8.0 kB       8.0 kB
  dist/static/font/inter-cyrillic-600-normal.8c69e1bb.woff2        8.0 kB       8.0 kB
  dist/static/font/inter-cyrillic-700-normal.9ce56ec3.woff2        8.0 kB       8.0 kB
  dist/static/font/inter-greek-100-normal.b849cf34.woff            9.8 kB       9.8 kB
  dist/static/font/inter-cyrillic-ext-100-normal.0e1aec1d.woff2    9.9 kB       9.9 kB
  dist/static/font/inter-greek-900-normal.8b67aaef.woff            9.9 kB       9.9 kB
  dist/static/font/inter-greek-400-normal.d6adbb78.woff            10.0 kB      10.0 kB
  dist/static/font/inter-cyrillic-100-normal.f9eff440.woff         10.0 kB      10.0 kB
  dist/static/font/inter-greek-800-normal.fe668711.woff            10.0 kB      10.0 kB
  dist/static/font/inter-greek-700-normal.384941e3.woff            10.0 kB      10.0 kB
  dist/static/font/inter-greek-300-normal.ec3d0adc.woff            10.0 kB      10.0 kB
  dist/static/font/inter-greek-500-normal.af596b86.woff            10.0 kB      10.0 kB
  dist/static/font/inter-greek-200-normal.299c0a99.woff            10.1 kB      10.1 kB
  dist/static/font/inter-greek-600-normal.61c756cf.woff            10.1 kB      10.1 kB
  dist/static/font/inter-cyrillic-400-normal.4cc6e28c.woff         10.1 kB      10.1 kB
  dist/static/font/inter-cyrillic-300-normal.edcd2385.woff         10.2 kB      10.2 kB
  dist/static/font/inter-cyrillic-900-normal.e0ed1158.woff         10.2 kB      10.2 kB
  dist/static/font/inter-cyrillic-ext-900-normal.b8d1a9c4.woff2    10.2 kB      10.2 kB
  dist/static/font/inter-cyrillic-ext-400-normal.2440d5f8.woff2    10.2 kB      10.2 kB
  dist/static/font/inter-cyrillic-200-normal.1ea11c46.woff         10.3 kB      10.3 kB
  dist/static/font/inter-cyrillic-ext-200-normal.43d8878a.woff2    10.3 kB      10.3 kB
  dist/static/font/inter-cyrillic-700-normal.571aeb62.woff         10.3 kB      10.3 kB
  dist/static/font/inter-cyrillic-500-normal.c1b1edeb.woff         10.3 kB      10.3 kB
  dist/static/font/inter-cyrillic-800-normal.9a8d829d.woff         10.3 kB      10.3 kB
  dist/static/font/inter-cyrillic-600-normal.c0105440.woff         10.3 kB      10.3 kB
  dist/static/font/inter-cyrillic-ext-300-normal.5df3b45f.woff2    10.3 kB      10.4 kB
  dist/static/font/inter-cyrillic-ext-800-normal.1c9e8f72.woff2    10.4 kB      10.5 kB
  dist/static/font/inter-cyrillic-ext-500-normal.d8f535fc.woff2    10.4 kB      10.5 kB
  dist/static/font/inter-cyrillic-ext-600-normal.dd95b020.woff2    10.5 kB      10.5 kB
  dist/static/font/inter-cyrillic-ext-700-normal.94d4fa7d.woff2    10.5 kB      10.5 kB
  dist/static/font/inter-cyrillic-ext-100-normal.191a6bdc.woff     13.9 kB      13.9 kB
  dist/static/font/inter-cyrillic-ext-900-normal.fefe92e2.woff     14.1 kB      14.1 kB
  dist/static/font/inter-cyrillic-ext-400-normal.6e13bad4.woff     14.2 kB      14.2 kB
  dist/static/font/inter-cyrillic-ext-300-normal.be7b9715.woff     14.3 kB      14.3 kB
  dist/static/font/inter-cyrillic-ext-200-normal.043206d9.woff     14.3 kB      14.3 kB
  dist/static/font/inter-cyrillic-ext-700-normal.9fa0c048.woff     14.3 kB      14.3 kB
  dist/static/font/inter-cyrillic-ext-500-normal.656d5a0e.woff     14.3 kB      14.3 kB
  dist/static/font/inter-cyrillic-ext-600-normal.62fe61a7.woff     14.4 kB      14.4 kB
  dist/static/font/inter-cyrillic-ext-800-normal.bf337f41.woff     14.4 kB      14.4 kB
  dist/static/font/inter-latin-100-normal.f52a4b98.woff2           23.4 kB      23.4 kB
  dist/static/font/inter-latin-400-normal.360a94a9.woff2           23.7 kB      23.7 kB
  dist/static/font/inter-latin-200-normal.b94fa4a8.woff2           24.0 kB      24.0 kB
  dist/static/font/inter-latin-900-normal.4c4b46c0.woff2           24.0 kB      24.0 kB
  dist/static/font/inter-latin-300-normal.0a506d8e.woff2           24.1 kB      24.1 kB
  dist/static/font/inter-latin-600-normal.efad9519.woff2           24.3 kB      24.3 kB
  dist/static/font/inter-latin-700-normal.6b51d3fc.woff2           24.4 kB      24.4 kB
  dist/static/font/inter-latin-500-normal.e98e390c.woff2           24.4 kB      24.4 kB
  dist/static/font/inter-latin-800-normal.ab3f402a.woff2           24.5 kB      24.5 kB
  dist/static/font/inter-latin-ext-100-normal.c40abd11.woff2       30.1 kB      30.1 kB
  dist/static/font/inter-latin-ext-400-normal.732723e2.woff2       30.7 kB      30.7 kB
  dist/static/font/inter-latin-100-normal.9da4947b.woff            30.9 kB      30.7 kB
  dist/static/font/inter-latin-ext-900-normal.46f24a46.woff2       31.0 kB      31.1 kB
  dist/static/font/inter-latin-400-normal.38abad60.woff            31.2 kB      31.0 kB
  dist/static/font/inter-latin-900-normal.db758083.woff            31.2 kB      31.0 kB
  dist/static/font/inter-latin-ext-200-normal.ccc0cca6.woff2       31.4 kB      31.4 kB
  dist/static/font/inter-latin-ext-300-normal.3033d32d.woff2       31.5 kB      31.5 kB
  dist/static/font/inter-latin-200-normal.2fb5efe4.woff            31.5 kB      31.3 kB
  dist/static/font/inter-latin-300-normal.38608292.woff            31.5 kB      31.3 kB
  dist/static/font/inter-latin-ext-500-normal.8f855dd9.woff2       31.8 kB      31.8 kB
  dist/static/font/inter-latin-500-normal.7986a549.woff            31.8 kB      31.6 kB
  dist/static/font/inter-latin-600-normal.8ad7b5a9.woff            31.8 kB      31.6 kB
  dist/static/font/inter-latin-800-normal.d1597229.woff            31.8 kB      31.6 kB
  dist/static/font/inter-latin-ext-700-normal.e8daf0b5.woff2       31.8 kB      31.8 kB
  dist/static/font/inter-latin-ext-800-normal.396ab648.woff2       31.8 kB      31.9 kB
  dist/static/font/inter-latin-700-normal.ac2885ce.woff            31.8 kB      31.6 kB
  dist/static/font/inter-latin-ext-600-normal.43dc1cee.woff2       32.0 kB      32.0 kB
  dist/static/js/lib-axios.f05593d5.js                             34.1 kB      13.4 kB
  dist/static/css/index.f288339e.css                               38.6 kB      8.3 kB
  dist/static/font/inter-latin-ext-100-normal.6e075b22.woff        42.4 kB      42.3 kB
  dist/static/font/inter-latin-ext-400-normal.d1f6a5a2.woff        42.9 kB      42.8 kB
  dist/static/font/inter-latin-ext-900-normal.11f54ef9.woff        42.9 kB      42.8 kB
  dist/static/font/inter-latin-ext-800-normal.0d8f8dff.woff        43.8 kB      43.7 kB
  dist/static/font/inter-latin-ext-500-normal.b3be213d.woff        43.8 kB      43.7 kB
  dist/static/font/inter-latin-ext-200-normal.954373b9.woff        43.9 kB      43.8 kB
  dist/static/font/inter-latin-ext-300-normal.9eef1747.woff        43.9 kB      43.8 kB
  dist/static/font/inter-latin-ext-700-normal.a2935e03.woff        43.9 kB      43.8 kB
  dist/static/font/inter-latin-ext-600-normal.8756e10d.woff        44.0 kB      43.8 kB
  dist/static/js/lib-react.41929aa2.js                             182.2 kB     57.8 kB
  dist/static/css/455.1d88f097.css                                 483.9 kB     284.5 kB
  dist/static/js/index.029e2c3c.js                                 840.6 kB     220.3 kB
  dist/static/js/455.47a866f0.js                                   6100.4 kB    1689.3 kB

  Total: 9618.2 kB (gzip: 4209.5 kB)


> build:electron:linux
> ./scripts/build-electron.sh linux

Building linux distribution

> bruno@2.0.0 dist:linux
> electron-builder --linux --x64 --dir --config electron-builder-config.js -c.electronDist=/usr/lib/electron33 -c.electronVersion=33.4.8

  • electron-builder  version=25.1.8 os=6.12.25-1-lts
  • loaded configuration  file=/home/bupd/.cache/yay/bruno/src/bruno-2.0.1/packages/bruno-electron/electron-builder-config.js
  • writing effective config  file=out/builder-effective-config.yaml
  • executing @electron/rebuild  electronVersion=33.4.8 arch=x64 buildFromSource=false appDir=./
  • installing native dependencies  arch=x64
  • preparing       moduleName=native-reg arch=x64
  • finished        moduleName=native-reg arch=x64
  • completed installing native dependencies
  • packaging       platform=linux arch=x64 electron=33.4.8 appOutDir=out/linux-unpacked
  • copying Electron  source=/usr/lib/electron33 destination=/home/bupd/.cache/yay/bruno/src/bruno-2.0.1/packages/bruno-electron/out/linux-unpacked
  • skipping "afterSign" hook as no signing occurred, perhaps you intended "afterPack"?
==> Entering fakeroot environment...
==> Starting package()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Stripping unneeded symbols from binaries and libraries...
objcopy: ./usr/lib/bruno/node_modules/native-reg/build/node_gyp_bins/python3: debuglink section already exists
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "bruno"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Creating package "bruno-debug"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: bruno 2.0.1-1 (Mon 28 Apr 2025 10:46:58 PM IST)
==> Cleaning up...
loading packages...
resolving dependencies...
looking for conflicting packages...
:: bruno-2.0.1-1 and bruno-bin-2.2.0-1 are in conflict. Remove bruno-bin? [y/N]
```

vbouchaud commented:
Ah, yes I understand. It would not have make a difference sadly: makepkg/yay/etc. would have downloaded the sources and build the package anyway. It is only when pacman is invoked for the installation that the conflict check is done.
